src/site/content/en/blog/receive-shared-data/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Receiving shared data
+subhead: Sharing on mobile made simple
+authors:
+  - petelepage
+  - joemedley
+date: 2019-11-08
+hero: hero.jpg
+alt: A hand sharing a web.dev article.
+description: |
+  On a mobile device, sharing should be as simple as clicking the Share button,
+  choosing an app, then choosing who to share with. The Web Share Target API
+  allows installed web apps to register with the underlying operating system to receive shared content.
+tags:
+  - post
+  - capabilities
+---
+
+On a mobile device, sharing should be as simple as clicking the **Share** button,
+choosing an app, and choosing who to share with. For example, you may want to
+share an interesting article, either by emailing it to friends or tweeting it to
+the world.
+
+In the past, only native apps could register with the operating system to
+receive shares from other installed apps. But with the Web Share Target API,
+installed web apps can register with the underlying operating system
+as a share target to receive shared content.
+Support for text and data was added in Chrome 71, and
+support for files was added in Chrome 76.
+
+{% Aside %}
+The Web Share Target API is only half of the magic. Web apps can share data,
+files, links, or text using the Web Share API. See
+[Share like a native](/share-like-a-native) for details.
```

Comment:
This link 404s. Is it another post in the capabilities collection that hasn't been published yet?

Reply:
Yes. It's #1829. We need to make sure these go live together.